javascripts/discourse/components/featured-homepage-topics.js

```diff
@@ -1,83 +1,116 @@
 import discourseComputed from "discourse-common/utils/decorators";
 import Component from "@ember/component";
+import I18n from "I18n";
 import { inject as service } from "@ember/service";
 import Topic from "discourse/models/topic";
 import { defaultHomepage } from "discourse/lib/utilities";
 
+const FEATURED_CLASS = "featured-homepage-topics";
+
 export default Component.extend({
   classNameBindings: ["featured-homepage-topics"],
   router: service(),
   titleElement: null,
   featuredTagTopics: null,
 
+  _getBanner() {
+    if (!(this.get("isDestroyed") || this.get("isDestroying"))) {
```

Comment:
No need for .get for unchained properties

Reply:
Yeah, the standard incantation is:

```suggestion
    if (this.isDestroying || this.isDestroyed) {
      return;
    }
```